src/sass/accordion/_base.scss

```diff
@@ -38,18 +38,32 @@
 
   &__toggle-text {
     color: $color-black-500;
-    font-size: $ts-18;
-    font-weight: $font-weight-bold;
+    font-size: $ts-20;
+    font-weight: $font-weight-regular;
     line-height: 1.3;
     padding-right: $spacing-sm;
   }
 
   &__toggle-icon {
     color: $color-black-400;
     margin-left: auto;
+    position: relative;
+    top: 4px;
   }
 
   &__panel {
-    padding: $spacing-md;
+    padding: $spacing-xs $spacing-sm $spacing-md $spacing-sm;
+  }
+
+  &__panel > * {
+    margin-top: 0;
```

Comment:
Instead of creating an extra rule set below with the `*:last-child block`, we can zero out the bottom margin here which is what we do in the `.rvt-prose` utility.

The point here being that we only apply margin/spacing to the _top_ of direct child elements to keep elements that might have both top and bottom margins from collapsing, causing inconsistent spacing. 

```diff
&__panel > * {
  margin-top: 0;
+ margin-bottom: 0;
}
```